### PR TITLE
Demo not working for users outside uberVU

### PR DESCRIPTION
**Pull request by RadValentin**
Made it so when no labels are provided, the plugin will show all issues.

cc: @skidding @catalinmiron

### CODE REVIEW

**Comment by RadValentin** [comment]
I've finished updating the demo, even added a nice gif to it :)
@skidding please review and let me know if there's anything else we can improve

File: js/changelog.js [hunk 1]
```diff
@@ -6,7 +6,7 @@
     listPosition: 'bottom',
     autoRefresh: false,
     githubRepo: 'uberVU/github-changelog-playground',
-    githubLabels: ['bug', 'enhancement', 'feature'],
+    githubLabels: [],
```
Comment (ovidiuch) [inline js/changelog.js]: The default should use labels, we'll just set githubLabels to [] in the demo

File: js/changelog.js [hunk 2]
```diff
@@ -122,16 +122,18 @@
...
           $update = $('<li>'),
-          $label = $('<span>', {class: CSS_PREFIX + '-label',
-                                text: featuredLabel.name,
-                                css: {
-                                  backgroundColor: '#' + featuredLabel.color
-                                }}),
+          $label,
           $title = $('<p>', {text: issue.title});
-
+      if(featuredLabel){
```
Comment (ovidiuch) [inline js/changelog.js]: Look at @NiGhTTraX's pull request to see how to style this correctly.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
+*.codekit
```
Comment (ovidiuch) [inline .gitignore]: As I said to @NiGhTTraX as well, I like to keep project-related rules in the project .gitignore and user-related rules in the [global gitignore](https://help.github.com/articles/ignoring-files#create-a-global-gitignore)

**Comment by ovidiuch** [comment]
Looking good.
1. Added some in-line comments
2. Remove the step with the labels from the demo
3. I'd add some more space between the two sections

File: js/changelog.js (hunk 1 above)
Comment (RadValentin) [inline js/changelog.js]: The reason I removed them was that if you initialized the plugin and set `githubLabels` to `[]` or `[bug]`(a single label) then the jQuery extend method would set it to `['bug', 'enhancement', 'feature']`. We can get around this by removing the deep extend [here](https://github.com/uberVU/github-changelog/blob/master/js/changelog.js#L32). This way whatever we pass in will override the default option.
What do you think?

**Comment by RadValentin** [comment]
Almost done here, just need to clarify things regarding `githubLabels`(see inline comment) and we should be good.

**Pull request merged**